packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -634,7 +619,7 @@ export class GarbageCollector implements IGarbageCollector {
 
 		// If the latest summary was updated but it was not tracked by this client, our state needs to be updated from
 		// this snapshot data.
-		if (this.shouldRunGC && result.latestSummaryUpdated && !result.wasSummaryTracked) {
+		if (this.configs.shouldRunGC && result.latestSummaryUpdated && !result.wasSummaryTracked) {
```

Comment:
Why change this?  Calls to `shouldRunGC` seem split almost 50-50 in this file as to whether they use the public getter or pull it off private configs. 🤷‍♂️